Service/MailerService.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace EMS\CoreBundle\Service;
+
+use Swift_Mailer;
+use Swift_Message;
+
+class MailerService
+{
+    const EMAIL_FROM = 'reporting@elasticms.test';
+    const NAME_FROM = 'ElasticMS';
```

Comment:
Can't be hard coded. Load this from the configuration (we already have email information in the .env)

Reply:
see:
EMS_FROM_EMAIL_ADDRESS='ems@test'
EMS_FROM_EMAIL_NAME='EMS'